WrathCombo/Combos/PvE/GNB/GNB.cs

```diff
@@ -162,24 +162,26 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
 
                     #region Mitigations
                     if (InCombat() && //Player is in combat
-                        IsPlayerTargeted() && //Player is being targeted by current target
                         !justMitted) //Player has not used a mitigation ability in the last 4-9 seconds
                     {
+                        if (IsPlayerTargeted())
+                        {
+                            //Nebula
+                            if (ActionReady(OriginalHook(Nebula)) && //Nebula is ready
+                                PlayerHealthPercentageHp() < 60) //Player's health is below 60%
+                                return OriginalHook(Nebula);
+
+                            //Rampart
+                            if (ActionReady(All.Rampart) && //Rampart is ready
+                                PlayerHealthPercentageHp() < 80) //Player's health is below 80%
+                                return All.Rampart;
+                        }
+
                         //Superbolide
                         if (ActionReady(Superbolide) && //Superbolide is ready
                             PlayerHealthPercentageHp() < 30) //Player's health is below 30%
                             return Superbolide;
 
```

Comment:
Probably best to put Superbolide to the top of the priority list here (at least until user priority lists are done)